### ISSUE TITLE
Hard tabs causing crash when embedded in a table

### ISSUE THREAD

**Issue by mattmook** [appmattus/markdown-lint#13]
The below markdown crashes the NoHardTabsRule with:

```
Caused by: java.lang.ArrayIndexOutOfBoundsException: 32
        at com.vladsch.flexmark.util.sequence.BasedSequenceImpl.indexOfAll(BasedSequenceImpl.java:1598)
        at com.appmattus.markdown.rules.NoHardTabsRule.visitDocument(NoHardTabsRule.kt:34)
```

```
# NoHardTabs failure

| Deep Link #1                  	| Deep Link #2                                  	| Additional Params 	| Screen 	| Activity 	| Notes 	|
|-------------------------------	|-----------------------------------------------	|-------------------	|--------	|----------	|-------	|
| Item 1                        	|                                               	|           -        	|        	|          	|       	|
| Item 2                          	|                                               	|           -        	|        	|          	|       	|
| Item 3                        	|                                               	|           -        	|        	|          	|       	|
| Item 4                        	|                                               	|           -        	|        	|          	|       	|
```

**Comment by mattmook** [comment]
This is caused by an issue in FlexMark https://github.com/vsch/flexmark-java/issues/362 where more than 32 matches cause the above exception to be thrown.

**Comment by mattmook** [comment]
Will be released in v0.4.2
